test/e2e/BUILD

```diff
@@ -210,20 +210,13 @@ go_library(
     ],
 )
 
-go_test(
+# phony go_default_test. Bazel test run interprets these as
```

Comment:
should these be go_binarys instead? our e2e infrastructure expects an `e2e_test` binary, I think.